packages/babel/src/preflightCheck.js

```diff
@@ -34,7 +34,8 @@ function helpersTestTransform() {
 const mismatchError = (actual, expected, filename) =>
   `You have declared using "${expected}" babelHelpers, but transforming ${filename} resulted in "${actual}". Please check your configuration.`;
 
-const inheritsHelperRe = /\/helpers\/(esm\/)?inherits/;
+// Revert to /\/helpers\/(esm\/)?inherits/ once https://github.com/babel/babel/issues/14185 is fixed
```

Comment:
```suggestion
// Revert to /\/helpers\/(esm\/)?inherits/ when Babel 8 gets released, this was fixed in https://github.com/babel/babel/issues/14185
```

Reply:
Since the problem turned out to be related to Babel itself we might want to keep this adjusted regex for now - given how popular Babel is we might as well go with this patch and end up being more friendly to users